src/mouse.c

```diff
@@ -294,25 +292,24 @@ void scrollMouse(int x, int y)
 
 #elif defined(IS_WINDOWS)
 
-	mouseScrollInputH.type = INPUT_MOUSE;
-	mouseScrollInputH.mi.dx = 0;
-	mouseScrollInputH.mi.dy = 0;
-	mouseScrollInputH.mi.dwFlags = MOUSEEVENTF_HWHEEL;
-	mouseScrollInputH.mi.time = 0;
-	mouseScrollInputH.mi.dwExtraInfo = 0;
+	mouseScrollInputs[0].type = INPUT_MOUSE;
+	mouseScrollInputs[0].mi.dx = 0;
+	mouseScrollInputs[0].mi.dy = 0;
+	mouseScrollInputs[0].mi.dwFlags = MOUSEEVENTF_HWHEEL;
+	mouseScrollInputs[0].mi.time = 0;
+	mouseScrollInputs[0].mi.dwExtraInfo = 0;
 	// Flip x to match other platforms.
-	mouseScrollInputH.mi.mouseData = -x;
-
-	mouseScrollInputV.type = INPUT_MOUSE;
-	mouseScrollInputV.mi.dx = 0;
-	mouseScrollInputV.mi.dy = 0;
-	mouseScrollInputV.mi.dwFlags = MOUSEEVENTF_WHEEL;
-	mouseScrollInputV.mi.time = 0;
-	mouseScrollInputV.mi.dwExtraInfo = 0;
-	mouseScrollInputV.mi.mouseData = y;
-
-	SendInput(1, &mouseScrollInputH, sizeof(mouseScrollInputH));
-	SendInput(1, &mouseScrollInputV, sizeof(mouseScrollInputV));
+	mouseScrollInputs[0].mi.mouseData = -x;
+
+	mouseScrollInputs[1].type = INPUT_MOUSE;
+	mouseScrollInputs[1].mi.dx = 0;
+	mouseScrollInputs[1].mi.dy = 0;
+	mouseScrollInputs[1].mi.dwFlags = MOUSEEVENTF_WHEEL;
+	mouseScrollInputs[1].mi.time = 0;
+	mouseScrollInputs[1].mi.dwExtraInfo = 0;
+	mouseScrollInputs[1].mi.mouseData = y;
+
+	SendInput(2, mouseScrollInputs, sizeof(mouseScrollInputs));
```

Comment:
This is incorrect, should be:

```c
SendInput(2, mouseScrollInputs, sizeof(mouseScrollInputs[0]));
```

Just spent a day to figure out why scrolling no longer work in v0.5.x on Windows. 😭